spec/features/users/intent_spec.rb

```diff
@@ -2,17 +2,37 @@
 require 'rails_helper'
 
 RSpec.feature 'Housing Intent' do
-  it 'can be indicated' do
-    student = FactoryGirl.create(:student_in_draw)
-    student.draw.update(status: 'pre_lottery')
-    log_in student
-    declare_off_campus student
-    expect(page).to have_css('.user-intent', text: 'off_campus')
+  context 'student' do
+    it 'can be indicated' do
+      student = FactoryGirl.create(:student_in_draw)
+      student.draw.update(status: 'pre_lottery')
+      log_in student
+      declare_off_campus student
+      expect(page).to have_css('.user-intent', text: 'off_campus')
+    end
+
+    def declare_off_campus(student)
+      visit "users/#{student.id}/intent"
+      select('off_campus', from: 'user_intent')
+      click_on 'Submit Intent'
+    end
   end
 
-  def declare_off_campus(student)
-    visit "users/#{student.id}/intent"
-    select('off_campus', from: 'user_intent')
-    click_on 'Submit Intent'
+  context 'admin' do
+    let!(:draw) { FactoryGirl.create(:draw, status: 'pre_lottery') }
+    it 'can lock intent changes on a per draw basis' do
+      student = FactoryGirl.create(:student, draw: draw)
+      lock_intent(draw)
+      log_in student
+      expect(page).not_to have_content('Update Housing Intent')
```

Comment:
I was going to ask where this link lives but then I realized it's in the nav. Maybe add a comment for future us' sake?